src/common/components/cards/how-to-check-text.tsx

```diff
@@ -54,7 +55,7 @@ export const HowToCheckText = NamedFC<HowToCheckTextProps>('HowToCheckText', pro
                         <li>
                             For detailed test instructions, see{' '}
                             <b>
-                                Assessment {'>'} Text legibility {'>'} Contrast
+                                Assessment {'>'} Adaptable content {'>'} Contrast
                             </b>
```

Comment:
I noticed this was previously in bold, so I left as is, but it seems that most of the bold styling is wrapped in <Markup.Term></Markup.Term>. Should this be changed to <Markup.Term> instead?

Reply:
Good catch! I agree that it would be an improvement to be consistent about using the semantically-named components; I think reusing `Term` would be better than using `<b>` here.